### PR TITLE
Add deploys endpoint

### PR DESCRIPTION
**Pull request by kucrut**
Accessible via `/wp-json/hm-stack/v1/deploys`.

### CODE REVIEW

File: plugins/hm-stack/inc/rest-controller/namespace.php [hunk 1]
```diff
@@ -307,3 +319,27 @@ function( $item ) {
...
+	$stack_data = API\get_deploys();
+
+	// If we've errored out, return a human-friendly message and code.
+	if ( is_wp_error( $stack_data ) ) {
+		// Prevent constant re-fetching in the event of a failure.
+		wp_cache_set( 'deploys', $stack_data, 'hm-stack', 5 * \MINUTE_IN_SECONDS );
+		return get_wp_error_for_hm_stack_return( $stack_data );
+	}
+
+	wp_cache_set( 'deploys', $stack_data, 'hm-stack', 12 * \HOUR_IN_SECONDS );
```
Comment (nathanielks) [inline plugins/hm-stack/inc/rest-controller/namespace.php]: Without knowing what this is being used for, I think it'd be in our best interest to lower this from 12 hours to something closer on a scale of minutes. As a user looking for a recent deploy, if I were to not see it in the list after hearing we just had one, I'd be suspect of something not working and reach out. To save us the support requests, it'd probably be good to lower this 😄

**Comment by roborourke** [comment]
Right you are :) @kucrut you ok to take this? Drop it to 1 minute maybe

**Comment by kucrut** [comment]
Addressed via https://github.com/humanmade/hm-platform/pull/119

**Comment by nathanielks** [comment]
💯

**Pull request merged**